DiamondLightSource/python-artemis
# 482 Added log handler to rollover tmp/dev/artemis.txt daily or when it reaches 100mb

Fixes #482

### To test:
1. Run tests, in particular in test_log.py 
2. Confirm tests don't create permanent txt files
